batch/batch/driver/main.py

```diff
@@ -455,7 +472,7 @@ async def pool_config_update(request, userdata):  # pylint: disable=unused-argum
 
     if not worker_local_ssd_data_disk:
         unreserved_disk_storage_gb = unreserved_worker_data_disk_size_gib(
-            worker_local_ssd_data_disk, worker_pd_ssd_data_disk_size_gb, worker_cores
+            pool.cloud, worker_local_ssd_data_disk, worker_pd_ssd_data_disk_size_gb, worker_cores
```

Comment:
See line 465 above. I'm worried the input for `worker_pd_ssd_data_disk_size_gb` here is 0 when there's a local ssd. This is also the value in the database. I'm not sure how you reconciled that it should be 375 later on in the code where is here it is 0.

Reply:
Ah, I see, sometimes we use PoolConfig to configure a pool and sometimes we use configure to configure a pool. I'll fix Pool.configure.

Your'e right that this function, `unreserved_worker_data_disk_size_gib` hasn't been updated to the new style yet. I'll do that in a subsequent PR.

Reply:
Ah, grr, this is annoying. The whole point was to centralize where we define ssd == 375, but since we use PoolConfig sometimes and configure other times, there are inherently two locations.

Reply:
OK, I consolidated all DB manipulation into PoolConfig. PoolConfig is basically our API to the database tables inst_colls and pools. Once the information comes out of the database, we create a Pool at which point we're just working with data_disk_size_gb.

Reply:
Good catch, this would've been a subtle bug to fix.